spec/rubocop/cop/vendor/rollbar_logger_spec.rb

```diff
@@ -38,11 +44,20 @@
     RUBY
   end
 
-  it 'autocorrect `Rollbar.debug` to `Rails.logger.debug` ' do
+  it 'autocorrect `Rollbar.debug(<string>)` to `Rails.logger.debug`' do
     expect(autocorrect_source('Rollbar.debug("Stale message")')).to eq('Rails.logger.debug("Stale message")')
   end
 
-  it 'autocorrect `Rollbar.info` to `Rails.logger.info` ' do
+  it 'autocorrect `Rollbar.info <string>` to `Rails.logger.info`' do
     expect(autocorrect_source('Rollbar.info "Stale message"')).to eq('Rails.logger.info "Stale message"')
   end
+
+  it 'autocorrect `Rollbar.debug(<hash>)` to `Rails.logger.debug`' do
+    expect(autocorrect_source('Rollbar.debug({foo: "bar"})')).to eq('Rails.logger.debug({foo: "bar"})')
```

Comment:
Could you check if this also works with `Rollbar.debug(foo: "bar")`? I believe it's a different AST.
If it doesn't and it's too hard to parse, we can leave it for now.

Reply:
Tested it and it works.